sdk/monitor/monitor-query/review/monitor-query.api.md

```diff
@@ -0,0 +1,313 @@
+## API Report File for "@azure/monitor-query"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import * as coreHttp from '@azure/core-http';
+import { OperationOptions } from '@azure/core-http';
+import { TokenCredential } from '@azure/core-auth';
+
+// @public
+export interface BatchRequest {
+    requests?: LogQueryRequest[];
+}
+
+// @public
+export interface BatchResponse {
+    error?: BatchResponseError;
+    responses?: LogQueryResponse[];
+}
+
+// @public
+export interface BatchResponseError {
+    code?: string;
+    // (undocumented)
+    innerError?: BatchResponseErrorInnerError;
+    message?: string;
+}
+
+// @public (undocumented)
+export interface BatchResponseErrorInnerError {
+    // (undocumented)
+    code?: string;
+    // (undocumented)
+    details?: ErrorDetails[];
+    // (undocumented)
+    message?: string;
+}
+
+// @public (undocumented)
+export interface ErrorDetails {
+    // (undocumented)
+    code?: string;
+    // (undocumented)
+    message?: string;
+    // (undocumented)
+    target?: string;
+}
+
+// @public
+export const last24Hours = "P1D";
+
+// @public
+export const last2Days = "P2D";
+
+// @public
+export const last30Minutes = "PT30M";
+
+// @public
+export const last3Days = "P3D";
+
+// @public
+export const last48Hours = "P2D";
+
+// @public
+export const last4Hours = "PT4H";
+
+// @public
+export const last5Minutes = "PT5M";
+
+// @public
+export const last7Days = "P7D";
+
+// @public
+export const lastDay = "P1D";
+
+// @public
+export const lastHour = "PT1H";
+
+// @public
+export interface LocalizableString {
+    localizedValue?: string;
+    value: string;
+}
+
+// @public
+export interface LogQueryRequest {
```

Comment:
this feels very strange. If we need to do something this lowlevel shouldn't we make a REST client instead?

Reply:
Agreed, I'll fix this up and probably just remove some of the lower level fields like `headers`, `method` and `path`.

Reply:
Fixed by not using it as the public model.

The newer one is simpler: https://github.com/richardpark-msft/azure-sdk-for-js/blob/edfc23f2f3c6854bbf254e3021c46cf593fa8638/sdk/monitor/monitor-query/src/models/logsModels.ts#L44